### PR TITLE
feat: added filters functionality in products page

### PR DESCRIPTION
**Pull request by ganesh-kanchi**

### CODE REVIEW

File: src/components/ProductFilters/ProductFilters.jsx [hunk 1]
```diff
@@ -1,46 +1,270 @@
...
-    <div class="filters">
-        <div class="filter-container">
-            <div class="filter-title">Price</div>
-            <div class="filter-value">
-                <div class="filter-price">
-                    <span>500</span>
-                    <span>5000</span>
+    <div className="filters">
+        <div className="filter-container">
+            <div className="filter-title">Price: &#8377;{productState.priceRange}</div>
```
Comment (t007rushi) [inline src/components/ProductFilters/ProductFilters.jsx]: use Icons instead

**Review by ajit-sharmaGH — APPROVED** [review]
Good variable naming, 
You can merge it, 
Code looks clean

File: src/index.js [hunk 2]
```diff
@@ -1,18 +1,32 @@
...
+// ReactDOM.render(
+//   <React.StrictMode>
+//     <Router>
+//       <ProductsProvider>
+//         <App/>
+//       </ProductsProvider>
+//     </Router>
+//   </React.StrictMode>,
+//   document.getElementById("root")
+// );
```
Comment (t007rushi) [inline src/index.js]: remove commeted code

File: src/pages/auth/Logout.jsx [hunk 3]
```diff
@@ -6,9 +6,9 @@ export const Logout = ()=>{
...
             <div className="card-component auth-card">
                 <div className="success-icon">
-                    <i class="fa-solid fa-check-circle"></i>
+                    <i className="fa-solid fa-check-circle"></i>
                 </div>
-                <div class="card-content heading-3">
+                <div className="card-content heading-3">
                     You have successfully been logged out!
                 </div>
                 <Link to={"/"}>Get back to Home</Link>
```
Comment (t007rushi) [inline src/pages/auth/Logout.jsx]: simply "/" no brace needed

**Review by t007rushi — APPROVED** [review]
try to have small PRs
work on suggestions 
good to merge

**Pull request merged**